tests/python/arith/test_arith_domain_touched.py

```diff
@@ -71,7 +71,8 @@ def test_domain_touched():
 def test_domain_touched_vector():
     m = tvm.runtime.convert(128)
 
-    @T.prim_func
+    # n is undefined
```

Comment:
Is this intended?

Reply:
Probably not.  We could make it well formed by adding `n: T.int32` to the arguments.  It looks like this test is validating that a fixed integer extent can be inferred, even when `n` is dynamic.